.circleci/config.yml

```diff
@@ -0,0 +1,155 @@
+              --disable-all \
+              --timeout 10m \
+              --enable gofmt \
+              --enable gosimple \
+              --enable govet
+      - notify_main_failure
+
+  # checks vendor directory is correct
+  check-vendor:
+    docker:
+      - image: *GOLANG_IMAGE
+    environment:
+      <<: *ENVIRONMENT
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "c6:96:98:82:dc:04:6c:39:dd:ac:83:05:e3:15:1c:98"
+      - checkout
+      - run:
+          command: go mod tidy
+      - run: |
+          if ! git diff --exit-code; then
+            echo "Git directory has vendor changes"
+            exit 1
+          fi
+      - notify_main_failure
+
+  go-test:
+    docker:
+      - image: *GOLANG_IMAGE
+      - image: docker.mirror.hashicorp.services/localstack/localstack:latest
+      - image: docker.mirror.hashicorp.services/letsencrypt/pebble
+        command: ["pebble"]
+        environment:
+          PEBBLE_VA_NOSLEEP: 1
+          PEBBLE_VA_ALWAYS_VALID: 1
+      - image: docker.mirror.hashicorp.services/circleci/postgres:11-alpine
+        environment:
+          POSTGRES_USER: postgres
+          POSTGRES_DB: waypoint_test
+      - image: docker.mirror.hashicorp.services/vault
+        command: ["server", "-dev", "-dev-root-token-id=hznroot"]
+    parallelism: 4
+    environment:
+      <<: *ENVIRONMENT
+      GOTAGS: ""
+      # GOMAXPROCS defaults to number of cores on underlying hardware, set
+      # explicitly to avoid OOM issues https://support.circleci.com/hc/en-us/articles/360034684273-common-GoLang-memory-issues
+      GOMAXPROCS: 4
+      # The medium resource class (default) boxes are 2 vCPUs, 4GB RAM
+      # https://circleci.com/docs/2.0/configuration-reference/#docker-executor
+      # but we can run a little over that limit.
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "c6:96:98:82:dc:04:6c:39:dd:ac:83:05:e3:15:1c:98"
+      - checkout
+      - attach_workspace:
+          at: /go/bin
+      - setup_remote_docker:
+          docker_layer_cache: true
+      - run: &install_gotestsum
+          name: Install gotestsum
+          command: |
+            url=https://github.com/gotestyourself/gotestsum/releases/download
+            curl -sSL "${url}/v${GOTESTSUM_RELEASE}/gotestsum_${GOTESTSUM_RELEASE}_linux_amd64.tar.gz" | \
+              sudo tar -xz --overwrite -C /usr/local/bin gotestsum
+      - run: go mod download
+      - run:
+          name: Waiting for Postgres to be ready
+          command: |
+            for i in `seq 1 10`;
+            do
+              nc -z localhost 5432 && echo Success && exit 0
+              echo -n .
+              sleep 1
+            done
+            echo Failed waiting for Postgres && exit 1
+      - run:
+          name: go test
+          command: |
+            mkdir -p $TEST_RESULTS_DIR
+            PACKAGE_NAMES=$(go list ./... | circleci tests split --split-by=timings --timings-type=classname)
```

Comment:
I'm getting an issue with not being able to run the `windows` specific stuff locally. Have you found a way around this?
```
package github.com/hashicorp/waypoint-plugin-sdk/internal/pkg/conpty
        imports golang.org/x/sys/windows: build constraints exclude all Go files in /home/rae/go/pkg/mod/golang.org/x/sys@v0.0.0-20200916030750-2334cc1a136f/windows
```

Reply:
I have not, although if I remember right I don't think that package works on windows? :thinking: @evanphx @mitchellh

Reply:
Fixed via https://github.com/hashicorp/waypoint-plugin-sdk/pull/18